study-builder/studies/pancan/about-you.conf

```diff
@@ -0,0 +1,82 @@
+{
+  include required("../snippets/activity-general-form.conf"),
+  "studyGuid": "cmi-pancan",
+  "activityCode": "ABOUTYOU",
```

Comment:
I recommend pulling out  identifiers into the `subs.conf` file, similar to how we're doing it for `testboston` and `rgp`.

Reply:
Its just a placeholder for now but will change it